### PR TITLE
Upgrade to support OpenShift 4.1 and K8S to 1.14

### PR DESCRIPTION
**Pull request by trinaths**
This PR addresses the update of CIS dependency packages to support OpenShift 4.1 and K8S 1.14

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,263 +1,121 @@
 {
 	"ImportPath": "github.com/F5Networks/k8s-bigip-ctlr",
-	"GoVersion": "go1.7",
+	"GoVersion": "go1.12",
```
[the remark this answers is not in the document]
Comment (ssurenr) [reply Godeps/Godeps.json]: https://github.com/F5Networks/k8s-bigip-ctlr/blob/4e267c80ca6769e1c65bea8c0d0ea3c4cd0ca5fd/build-tools/Dockerfile.debian.builder#L17
https://github.com/F5Networks/k8s-bigip-ctlr/blob/4e267c80ca6769e1c65bea8c0d0ea3c4cd0ca5fd/build-tools/Dockerfile.rhel7.builder#L5

File: build-tools/attributions-generator.sh [hunk 2]
```diff
@@ -7,7 +7,8 @@
 set -e
 set -x
 
-ATTR_GEN_IMG=f5networksdevel/attributions-generator:latest
+#ATTR_GEN_IMG=f5networksdevel/attributions-generator:latest
```
[the remark this answers is not in the document]
Comment (trinaths) [reply build-tools/attributions-generator.sh]: Done.

File: Godeps/Godeps.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (trinaths) [reply Godeps/Godeps.json]: Done. Update the SHA too

File: build-tools/Dockerfile.rhel7.builder [hunk 3]
```diff
@@ -2,7 +2,7 @@ FROM registry.redhat.io/rhel7
 
 # GOLANG install steps
 
-ENV GOLANG_VERSION 1.11.1
+ENV GOLANG_VERSION 1.12
 ENV GOLANG_SRC_URL https://golang.org/dl/go$GOLANG_VERSION.src.tar.gz
 ENV GOLANG_SRC_SHA256 558f8c169ae215e25b81421596e8de7572bd3ba824b79add22fba6e284db1117
```
Comment (ssurenr) [inline build-tools/Dockerfile.rhel7.builder]: We need to update this SHA

Comment (trinaths) [reply build-tools/Dockerfile.rhel7.builder]: Done

**Pull request merged**